src/App.jsx

```diff
@@ -1,3 +1,30 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import PrivateRoute from './components/PrivateRoute';
+import Header from './views/Header';
+import Home from './views/Home';
+import Guestbook from './views/Guestbook';
+import Authentication from './views/Authentication';
+import styles from './App.css';
+import { UserProvider } from './context/UserContext';
+
 export default function App() {
-  return <h1>Hello World</h1>;
+  return (
+    <main className={styles.main}>
+      <UserProvider>
```

Comment:
I think you will have an easier time testing if you put your `UserProvider` in `index.js` instead of `App.jsx`